tests/result/lisp_registration.pcap.out

```diff
@@ -0,0 +1,17 @@
+Guessed flow protos:	2
+
+DPI Packets (TCP):	14	(7.00 pkts/flow)
+DPI Packets (UDP):	2	(1.00 pkts/flow)
+Confidence Unknown          : 2 (flows)
+Confidence DPI              : 2 (flows)
+
+Unknown	14	2926	2
+LISP	16	2340	2
+
+	1	UDP 10.0.123.3:4342 <-> 10.0.123.1:4342 [proto: 236/LISP][ClearText][Confidence: DPI][cat: Cloud/13][4 pkts/636 bytes <-> 4 pkts/568 bytes][Goodput ratio: 73/70][0.01 sec][bytes ratio: 0.056 (Mixed)][IAT c2s/s2c min/avg/max/stddev: 0/0 0/2 0/5 0/2][Pkt Len c2s/s2c min/avg/max/stddev: 142/130 159/142 182/154 15/8][Plen Bins: 0,0,12,75,12,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+	2	UDP 10.0.123.2:4342 <-> 10.0.123.1:4342 [proto: 236/LISP][ClearText][Confidence: DPI][cat: Cloud/13][4 pkts/568 bytes <-> 4 pkts/568 bytes][Goodput ratio: 70/70][0.01 sec][bytes ratio: 0.000 (Mixed)][IAT c2s/s2c min/avg/max/stddev: 0/3 0/1 0/3 0/1][Pkt Len c2s/s2c min/avg/max/stddev: 130/130 142/142 154/154 8/8][Plen Bins: 0,0,25,75,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+
+
+Undetected flows:
+	1	TCP 10.0.123.3:52995 <-> 10.0.123.1:4342 [proto: 0/Unknown][ClearText][Confidence: Unknown][4 pkts/894 bytes <-> 3 pkts/715 bytes][Goodput ratio: 74/76][0.36 sec][bytes ratio: 0.111 (Mixed)][IAT c2s/s2c min/avg/max/stddev: 1/1 120/73 213/145 88/72][Pkt Len c2s/s2c min/avg/max/stddev: 60/60 224/238 714/586 283/246][Plen Bins: 33,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,33,0,0,0,33,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
```

Comment:
Those flows may be related to LISP as well?

Reply:
Another think I'll take a look into...

Reply:
Fixed in last push